ui/apps/dev-server-ui/src/components/Event/SendEventModal.tsx

```diff
@@ -200,12 +200,13 @@ export default function SendEventModal({ data, isOpen, onClose }: SendEventModal
       description="Send an event manually by filling or pasting a payload"
       className="w-full max-w-5xl"
       footer={
-        <div className="flex items-center justify-between">
-          <Button label="Cancel" appearance="outlined" btnAction={onClose} />
-          <Button
+        <div className="flex items-center justify-end gap-2">
+          <NewButton kind="secondary" label="Cancel" appearance="outlined" onClick={onClose} />
+          <NewButton
+            kind="primary"
             disabled={sendEventState.isLoading}
             label="Send Event"
-            btnAction={() => sendEvent()}
+            onClick={() => sendEvent()}
```

Comment:
oh thank goodness for `btnAction` being in the past!

Reply:
👍